news/2018-07-26-weld-305Final.asciidoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: news
+title: Weld 3.0.5.Final
+author: Matej Novotny
+priority: 1
+change_frequency: weekly
+excerpt: Weld 3.0.5.Final released
+desc: Weld 3.0.5.Final released
+tags: [release]
+---
+:linkattrs:
+
+Weld 3.0.5.Final is here and hand in hand with it comes Weld API 3.0.SP4.
+And in case you missed it, there was link:https://search.maven.org/#artifactdetails%7Cjavax.enterprise%7Ccdi-api%7C2.0.SP1%7Cjar[CDI 2.0.SP1 release] as well.
+It comes with few link:https://issues.jboss.org/issues/?jql=project%20%3D%20CDI%20AND%20fixVersion%20%3D%202.0.SP1[security manager related fixes] and Weld is of course compliant with it.
+If you are running on a servlet or in SE, you may need to update it yourself, in case of EE it usually falls to the server to update this dependency for you.
+
+NOTE: This time around we are not providing a patch for WildFly 13 as there were major changes blocking us from doing so.
+However, Wildfly 14 will contain all of the above updates - Weld core, Weld API, CDI API - so keep an eye out for its release!
+
+As for updates and fixes, we keep looking at JDK 11 and have done another round of fixing to keep up with adjustments and removals going on in JDK.
+There is was a bunch of interceptor/decorator fixes for those of you who indulge in using complex class hierarchies with abstractions, generics and default methods.
```

Comment:
There is was -> There is  ?

Reply:
@manovotn A second attempt ;-) https://github.com/weld/weld.github.io/pull/64#discussion_r205130027